### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>HEADER</header>
+    <footer>Footer</footer>
+</body>
+</html>
```
Comment (loknathPradhan) [inline index.html:16]: you should make a div

Comment (rptomar) [inline index.html:16]: you should make a saparate div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
```
Comment (Sarhan786) [inline index.html:9]: wrape the header and footer in div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <header>HEADER</header>
```
Comment (vineetha012) [inline index.html]: place  header and footer inside one div

Comment (AvinashKalwale) [inline index.html]: looks good

Comment (JenniferJoseph98) [inline index.html]: add div tag

Comment (AvinashKalwale) [reply index.html]: use DIV

**Pull request merged**